docs/content/rest-api.md

```diff
@@ -5,6 +5,12 @@ weight: 80
 restrictedtoc: true
 ---
 
+{{< info >}}
+Integrating with OPA from a programming language? You might find it easier to build your
+OPA integration using one of the [language SDKs](/ecosystem/#languages) than working
+with the REST API directly.
+{{< /info >}}
+
```

Comment:
I find this note a bit misplaced. I mean we're on the rest api page and right at the top we're hinting that this approach may not be ideal. I would rather have this somewhere in the text later on or at the end imo.

Reply:
The point is to ensure that users are aware that they *dont* have to deal with the REST API unless they need to.

Reply:
I've opted to move this down to the end of the intro, it still features prominently above the fold.